src/js/state/createGlobalStore.ts

```diff
@@ -2,5 +2,15 @@ import {configureStore} from "@reduxjs/toolkit"
 import globalReducer, {GlobalState} from "./globalReducer"
 
 export default function(initState: GlobalState | undefined) {
-  return configureStore({reducer: globalReducer, preloadedState: initState})
+  return configureStore({
+    reducer: globalReducer,
+    preloadedState: initState,
+    middleware: (getDefaultMiddleware) => {
+      return getDefaultMiddleware({
+        thunk: true,
+        immutableCheck: false,
```

Comment:
These checks seem useful, can we turn them on or do we have immutability/serializable issues still?

Reply:
I was just getting warnings when running the tests that the middleware was taking over 30ms and should maybe be disabled. So I disabled it.